Formula/csound.rb

```diff
@@ -14,7 +14,9 @@ class Csound < Formula
   depends_on "cmake" => :build
   depends_on "python" => [:build, :test]
   depends_on "python@2" => [:build, :test]
+  depends_on "swig" => :build
   depends_on "fltk"
+  depends_on :java
```

Comment:
Is this always needed? Or could this just be like python?

Reply:
It isn’t always needed. I’ve changed this to `depends_on :java => [:build, :test]`.